src/web-socket-handler_test.ts

```diff
@@ -2,6 +2,7 @@ import { promisify } from 'util';
 import { expect } from 'chai';
 import WebSocket = require('isomorphic-ws');
 import { ReadableStreamBuffer, WritableStreamBuffer } from 'stream-buffers';
+import stream = require('stream');
```

Comment:
```suggestion
import stream from 'node:stream';
```

Reply:
Converted all to `readUint8` The Internet seems to be ambiguous on which is the right casing. Both appear to work...